proposals/devcontainer-features.md

```diff
@@ -65,6 +25,7 @@ The properties of the file are as follows:
 | Property | Type | Description |
 | :--- | :--- | :--- |
 | id | string | Id of the feature/definition. The id should be unique in the context of the repository/published package where the feature exists. |
+| version | string | The semantic version of the feature. |
```

Comment:
Duplicate: on L:33 there's already a `version` property, maybe we'd like to update the description instead?

Reply:
Thanks, good catch!